### PR TITLE
FRONTEND Vplan edited into VPTOOL

### PR DESCRIPTION
**Pull request by JeanRochCoulon**
Please clone the project and execute cva6/docs/VerifPlans/FRONTEND/runme.sh to see the FRONTEND verif plan.
This is a draft version, VPTOOL should be extended to support yaml database. And Verif plan should be reviewed by the team (@ASintzoff, @mike, etc...)

### CODE REVIEW

File: cva6/docs/VerifPlans/FRONTEND/runme.sh [hunk 1]
```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
```
Comment (MikeOpenHWGroup) [inline cva6/docs/VerifPlans/FRONTEND/runme.sh:1]: Please add a one-line SPDX header.

**Review by MikeOpenHWGroup — CHANGES_REQUESTED** [review]
Please add an SPDX header to all new files.   Is it possible to do this with `pck` files?

File: cva6/docs/VerifPlans/FRONTEND/runme.sh (hunk 1 above)
Comment (MikeOpenHWGroup) [reply cva6/docs/VerifPlans/FRONTEND/runme.sh:1]: Understood.

**Review by zchamski — COMMENTED** [review]
Just add the SPDX header on the runme.sh file, otherwise LGTM

**Review by MikeOpenHWGroup — APPROVED** [review]

**Comment by MikeOpenHWGroup** [comment]
LGTM.   I will leave the final approval and merge to @JeanRochCoulon

**Review by MikeOpenHWGroup — APPROVED** [review]

File: cva6/docs/VerifPlans/source/dvplan_intro.rst [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+* **[RVpriv]** “The RISC-V Instruction Set Manual, Volume II: Privileged Architecture, Document Version 20211203”, Editors Andrew Waterman, Krste Asanović and John Hauser, RISC-V Foundation, December 4, 2021.
+* **[RVdbg]** “RISC-V External Debug Support, Document Version 0.13.2”, Editors Tim Newsome and Megan Wachs, RISC-V Foundation, March 22, 2019.
+* **[RVcompat]** “RISC-V Architectural Compatibility Test Framework”, https://github.com/riscv-non-isa/riscv-arch-test.
+* **[AXI]** AXI Specification, https://developer.arm.com/documentation/ihi0022/hc.
+* **[CV-X-IF]** Placeholder for the CV-X-IF coprocessor interface currently prepared at OpenHW Group; current version in https://docs.openhwgroup.org/projects/openhw-group-core-v-xif/.
+* **[OpenPiton]** “OpenPiton Microarchitecture Specification”, Princeton University, https://parallel.princeton.edu/openpiton/docs/micro_arch.pdf.
+
+CV32A6 is a standards-compliant 32-bit processor fully compliant with RISC-V specifications: [RVunpriv], [RVpriv] and [RVdbg] and passes [RVcompat] compatibility tests, as requested by [GEN-10] in [CVA6req].
```
Comment (ASintzoff) [inline cva6/docs/VerifPlans/source/dvplan_intro.rst:50]: a standards-compliant 32-bit processor => a 32-bit processor

File: cva6/docs/VerifPlans/source/dvplan_intro.rst [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+Getting start verification
+--------------------------
+
+[TO BE COMPLETED]
+
+
+Documentation framework
+-----------------------
+
+The framework of this document is aligned on the CVA6 design document [CVA6design].
```
Comment (ASintzoff) [inline cva6/docs/VerifPlans/source/dvplan_intro.rst:62]: aligned on => aligned with

File: cva6/docs/VerifPlans/source/dvplan_intro.rst [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+
+Description of the framework:
+
+* Processor is a subsystem
+* Processor subsystems is split into several modules
+* Module are verified separately
```
Comment (ASintzoff) [inline cva6/docs/VerifPlans/source/dvplan_intro.rst:68]: Module are => Modules are

File: cva6/docs/VerifPlans/source/dvplan_intro.rst [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+
+Documentation framework
+-----------------------
+
+The framework of this document is aligned on the CVA6 design document [CVA6design].
+
+Description of the framework:
+
+* Processor is a subsystem
+* Processor subsystems is split into several modules
```
Comment (ASintzoff) [inline cva6/docs/VerifPlans/source/dvplan_intro.rst:67]: subsystems is => subsystems are

**Comment by JeanRochCoulon** [comment]
Thanks to everyone for feedbacks, it is time to merge !

**Pull request merged**